packages/reactivity/__tests__/computed.spec.ts

```diff
@@ -1012,6 +1012,42 @@ describe('reactivity/computed', () => {
     expect(cValue.value).toBe(1)
   })
 
+  test('should not recompute when condition changes if computed does not track reactive data', async () => {
```

Comment:
This could be simplified as:

```ts
  test('', () => {
    const spy = vi.fn()
    const c1 = computed(() => spy())

    c1.value
    ref(0).value++ // update globalVersion
    c1.value

    expect(spy).toBeCalledTimes(1)
  })
  ```

Reply:
Nice.